[0m19:12:41.732680 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c2142f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c715130>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c714da0>]}


============================== 19:12:41.740040 | 000a935f-c3cd-4bf2-9389-66db1aa47a71 ==============================
[0m19:12:41.740040 [info ] [MainThread]: Running with dbt=1.9.0
[0m19:12:41.740824 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/flerinv/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt init dbt_scooters', 'send_anonymous_usage_stats': 'True'}
[0m19:12:41.770171 [debug] [MainThread]: Starter project path: /opt/anaconda3/lib/python3.12/site-packages/dbt/include/starter_project
[0m19:12:41.783206 [info ] [MainThread]: 
Your new dbt project "dbt_scooters" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m19:12:41.783910 [info ] [MainThread]: Setting up your profile.
[0m19:15:27.570821 [info ] [MainThread]: Profile dbt_scooters written to /Users/flerinv/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m19:15:27.578038 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 165.92365, "process_in_blocks": "0", "process_kernel_time": 0.518342, "process_mem_max_rss": "97734656", "process_out_blocks": "0", "process_user_time": 1.960099}
[0m19:15:27.578887 [debug] [MainThread]: Command `dbt init` succeeded at 19:15:27.578681 after 165.92 seconds
[0m19:15:27.579566 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c715250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c59ede0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10c36b380>]}
[0m19:15:27.580177 [debug] [MainThread]: Flushing usage events
[0m19:15:28.239969 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:31:41.628929 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108ce1250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1091cd550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1091cd1c0>]}


============================== 19:31:41.636865 | 86b25d89-0e7b-4328-bb0a-fdfb24fca212 ==============================
[0m19:31:41.636865 [info ] [MainThread]: Running with dbt=1.9.0
[0m19:31:41.638182 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': '/Users/flerinv/.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt compile', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m19:31:41.639158 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/flerinv/Desktop/db/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m19:31:41.644102 [debug] [MainThread]: Resource report: {"command_name": "compile", "command_success": false, "command_wall_clock_time": 0.11682542, "process_in_blocks": "0", "process_kernel_time": 0.44756, "process_mem_max_rss": "94085120", "process_out_blocks": "0", "process_user_time": 1.788362}
[0m19:31:41.644916 [debug] [MainThread]: Command `dbt compile` failed at 19:31:41.644729 after 0.12 seconds
[0m19:31:41.645335 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1091cd400>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1091cd6a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1091cee10>]}
[0m19:31:41.645846 [debug] [MainThread]: Flushing usage events
[0m19:31:42.239022 [debug] [MainThread]: An error was encountered while trying to flush usage events
